process/utility/utils_test.go

```diff
@@ -256,3 +257,73 @@ func TestEncode_BlockResult(t *testing.T) {
 	_, err = testAvroMarshaller.Encode(&blockResNilBlock)
 	require.NotNil(t, err)
 }
+
+func TestUInt32SliceToInt32Slice(t *testing.T) {
+	t.Parallel()
+
+	out := utility.UInt32SliceToInt32Slice(nil)
+	require.Equal(t, []int32{}, out)
+
+	out = utility.UInt32SliceToInt32Slice([]uint32{44, 555})
+	require.Equal(t, []int32{44, 555}, out)
+}
+
+func TestGetBigIntBytesFromStr(t *testing.T) {
+	t.Parallel()
+
+	t.Run("should work", func(t *testing.T) {
+		t.Parallel()
+
+		ret, err := utility.GetBigIntBytesFromStr("4321")
+		require.Equal(t, big.NewInt(4321).Bytes(), ret)
+		require.Nil(t, err)
+	})
+
+	t.Run("empty value, should return bigInt(0)", func(t *testing.T) {
+		t.Parallel()
+
+		ret, err := utility.GetBigIntBytesFromStr("")
+		require.Equal(t, big.NewInt(0).Bytes(), ret)
+		require.Nil(t, err)
+	})
+
+	t.Run("invalid value in base 10", func(t *testing.T) {
+		t.Parallel()
+
+		ret, err := utility.GetBigIntBytesFromStr("ff")
+		require.Nil(t, ret)
+		require.NotNil(t, err)
```

Comment:
can you check here for `errInvalidValueInBase10`

Reply:
I wanted to do this in the first place, but had to export the error variable .
Done it as requested